### PR TITLE
Add OmniOS buildtools

### PR DESCRIPTION
**Pull request by HeinrichHartmann**
Allow for to control libmtev build from a remote omnios host
Add librabbitmq dependency

### CODE REVIEW

File: buildtools/omnios/configure.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+autoreconf -i
+
+./configure \
+  --prefix=/opt/circonus/ \
+  --exec-prefix=/opt/circonus \
+  --sysconfdir=/opt/circonus/etc \
+  --includedir=/opt/circonus/include \
```
Comment (postwait) [inline buildtools/omnios/configure.sh]: Shouldn't this be /opt/circonus/include/amd64 ?

Comment (HeinrichHartmann) [inline buildtools/omnios/configure.sh]: I copied this over from snowh. But I can change it if needed.

**Comment by HeinrichHartmann** [comment]
This is a circonus-labs repo, so I don't see anything wrong with us using `/opt/criconus` paths for our development.

If we are going to support Vagrant / Docker builds in the future (I think we should) build information is needed inside this repo. Packaging could then use these scrpts as well.

Even if we don't use them elsewhere and they run out of date, then they are more likely to be working than copy/paste from the readme.

**Comment by HeinrichHartmann** [comment]
Using --prefix=/usr/local breaks mtevbusted since it expects luamtev in /opt/circonus:
https://github.com/circonus-labs/libmtev/blob/master/test/mtevbusted-script.in#L1

Maybe that should be fixed.

**Comment by postwait** [comment]
That's a bug in mtevbusted.. it should respect the install prefix.

**Comment by postwait** [comment]
Circonus specific build stuff should live in a different tree.

**Pull request closed without merging**